x-pack/plugins/maps/server/tutorials/ems/index.ts

```diff
@@ -60,7 +60,7 @@ Indexing EMS administrative boundaries in Elasticsearch allows for search on bou
                   }),
                   textPre: i18n.translate('xpack.maps.tutorials.ems.uploadStepText', {
                     defaultMessage:
-                      '1. Open [Elastic Maps]({newMapUrl}).\n\
+                      '1. Open [Maps]({newMapUrl}).\n\
```

Comment:
Slipped this text change in. Maps is know known as `Maps` instead of `Elastic Maps`

Reply:
yup!